pkg/cmdutil/userio/multi_select.go

```diff
@@ -197,11 +196,15 @@ func (d multiSelectItemDelegate) Render(w io.Writer, m list.Model, index int, li
 	} else {
 		checkedSign = "[ ] "
 	}
-	str := checkedSign + strconv.Itoa(index+1) + ". " + it.value
+	str := checkedSign + it.value
 
 	if index == m.Index() {
-		_, _ = fmt.Fprint(w, d.styles.selectedItem.Render(str))
+		_, _ = fmt.Fprint(w, d.styles.selectedItem.Render("> "+str))
 	} else {
+		// NB: The "item" style already renders with a padding of 2 spaces on
```

Comment:
nitpicking, but shouldn't that block comment be above the else?

Reply:
Yes, good point. I reworked that.